src/models/Offer.js

```diff
@@ -18,37 +18,54 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 
+type ExchangeOfferAsset = {
```

Comment:
This could be just `OfferAsset` since doesn't look like something specific to exchange. What do you think?

Reply:
Hm, yes and no :)
This is a type that is specifically used in exchange offer. I'd suggest leaving it as is 🤔